reana_workflow_controller/workflow_run_manager.py

```diff
@@ -126,6 +127,13 @@ def start_interactive_session(self):
         """Start an interactive workflow run."""
         raise NotImplementedError('')
 
+    def stop_interactive_session(self):
```

Comment:
Let's move this implementation to `KubernetesWorkflowRunManager` since it is Kubernetes specific and leave a not implemented here.